vcbuild.bat

```diff
@@ -68,7 +68,7 @@ if /i "%1"=="test-pummel"   set test_args=%test_args% pummel&goto arg-ok
 if /i "%1"=="test-all"      set test_args=%test_args% sequential parallel message gc internet pummel&set buildnodeweak=1&set jslint=1&goto arg-ok
 if /i "%1"=="test-known-issues" set test_args=%test_args% known_issues --expect-fail&goto arg-ok
 if /i "%1"=="jslint"        set jslint=1&goto arg-ok
-if /i "%1"=="package"       set package=1&goto arg-ok
+if /i "%1"=="package"       set package=1&set i18n_arg=small-icu&goto arg-ok
```

Comment:
Please could you also `set download_arg="--download=all"` (also for the `dll` option) as v4.x does not contain an in-tree copy of ICU.

Reply:
originally it was desired to not automatically download without the user opting in

Reply:
Fair point. What happens though if `small-icu` is used where there is no local ICU?

Reply:
To answer my own question:
```
Looking for Visual Studio 2015
Found Visual Studio 2015
configure  --with-intl=small-icu --dest-cpu=x64 --tag=
ctrpp not found in WinSDK path--using pre-gen files from tools/msvs/genfiles.
creating icu_config.gypi
* ECMA-402 (Intl) support didn't find ICU in deps\icu..
Warning: Not downloading package "icu". You could pass "--download=all"
    (Windows: "download-all") to try auto-downloading it.
 Cannot build Intl without ICU in deps\icu.
 (Fix, or disable with "--with-intl=none" )
Failed to create vc project files.
```

So for v4.x it doesn't seem to make sense to specify `set i18n_arg=small-icu` without also specifying `set download_arg="--download=all"`. Note this is currently done for the `msi` option and I think `package` and `dll` should do the same for consistency: 

https://github.com/nodejs/node/blob/920699c5053447ef5888243bdff67b7f8effbeb9/vcbuild.bat#L72